test/Conversion/FIRRTLToRTL/lower-to-rtl.mlir

```diff
@@ -212,10 +214,12 @@ firrtl.circuit "Simple" {
     %c306_ui10 = firrtl.constant(306 : ui10) : !firrtl.uint<10>
     %50 = firrtl.div %c104_ui8, %c306_ui10 : (!firrtl.uint<8>, !firrtl.uint<10>) -> !firrtl.uint<8>
 
-    // Issue #364: https://github.com/llvm/circt/issues/364
-    // CHECK: = rtl.constant -873 : i12
     %c1175_ui11 = firrtl.constant(1175 : ui11) : !firrtl.uint<11>
     %51 = firrtl.neg %c1175_ui11 : (!firrtl.uint<11>) -> !firrtl.sint<12>
+    // https://github.com/llvm/circt/issues/821
+    // CHECK:  = comb.concat %false, %in1 : (i1, i4) -> i5
+    // CHECK:  = comb.sub %c0_i5, %65 : i5
+    %52 = firrtl.neg %in1 : (!firrtl.uint<4>) -> !firrtl.sint<5>
```

Comment:
Since it is missing, please add the firrtl.neg (sint) and firrtl.neg(sint<0>) and firrtl.neg(uint<0>)